### DISCUSSION TITLE
Kindle firmware version recommendation for koreader

### DISCUSSION THREAD

**Discussion by MartletH** [koreader/koreader#8259] [Show and tell]
I've tried several kindle firmwares with KPW2, here's a one sentense recommendation.
Use **KPW2 kindle firmware 5.7.x~5.8.7**
Thanks to jimman2003 **KPW4 firmware 5.13.5**
Really appreciate the nice work of koreader.

Details,
* Firmware 5.4.3.2, flashes home screen contents from time to time
* Firmware 5.6.5, flashes home screen at sleep
* Firmware 5.7.4, flashes small part of home screen contents (the book picture) only with start koreader ASAP
* Firmware 5.8.7, not tried, it is the highest firmware with legacy jaibreak method
* Firmware 5.10.3, no flash home screen, even with start koreader ASAP, but the jailbreak is not stable, crashes with wifi connected

**Comment by Galunid** [comment]
So I used 5.7.4 I believe, and didn't have trouble at all, but I used KUAL and start KOReader (not ASAP). The blinking were an issue when I started KOReader using KPVbooklet (by clicking epub book). It's been a really long time since I used kindle though, so ymmv

**Reply by MartletH** [reply]
Blink to blank? This is kpvbooklet issue, I can't remember where mentioned, just drop this plugin until it's fixed.
Kindle is my favorite reader since touch version. Koreader makes it better on pdf and epub.

**Comment by jimman2003** [comment]
I run KOReader on a KPW4 , firmware verison 5.13.5 , no problems at all

**Reply by MartletH** [reply]
Thank you, I'll consolidate for everyone's convenience.
